src/components/TooltipWrapper/TooltipWrapper.tsx

```diff
@@ -0,0 +1,21 @@
+import { Tooltip } from '@patternfly/react-core';
+import { InfoCircleIcon } from '@patternfly/react-icons';
+
+interface ITooltipWrapperProps {
+  tooltip: string;
+}
+/**
+ * Tooltip Wrapper to make any component (like icon) to become a tooltip component
+ *
+ * @param tooltip - the tooltip to be displayed
+ * @param children - the tooltip handler, by default <InfoCircleIcon />
+ */
+export const TooltipWrapper = ({ tooltip, children }: React.PropsWithChildren<ITooltipWrapperProps>) => {
+  return (
+    <span>
+      <Tooltip removeFindDomNode content={tooltip}>
+        <> {children ? <>{children}</> : <InfoCircleIcon />}</>
```

Comment:
outside one was removed, but the other one is required. 
![image](https://user-images.githubusercontent.com/6870455/223421713-99d5e582-61ed-4256-8c50-f172d0ce52f3.png)

Reply:
the outside one can not be removed either, because it is used to include blank space to the icon. 
![image](https://user-images.githubusercontent.com/6870455/223459398-f6505ca8-f289-43d7-9afb-e1e633346053.png)